pyproject.toml

```diff
@@ -40,6 +40,7 @@ dependencies = [
     "pyyaml>=0.0.0",
     "requests>=0.0.0",
     "openpyxl>=0.0.0",
+    "nominal-streaming>=0.5.7",
```

Comment:
if this is a required dependency, then i guess we don't need to do the import warning at all?

also this is in the "any version" section of the deps, should move it up i guess